### ISSUE TITLE
Commit body indicator in the log

### ISSUE THREAD

**Issue by jazzzz** [magit/magit#4127]
It would be nice to have an indicator for each commit with a body in the message, like in GitHub here: ![image](https://user-images.githubusercontent.com/290626/82930212-4d9fb380-9f85-11ea-888d-fc363242e65b.png)
It would allow to see directly which commits have more than a subject.

**Comment by tarsius** [comment]
Personally I feel like this feature would not be much of an improvement. (Or worse would make things slightly worse by wasting some space.) So that's one reason why that feature doesn't exist.

The much more important reason however is that it would make parsing of the git output more difficult and more expensive. We would have to use `%b` in `log --format` and then we would have to move over all those lines just so we can know whether there are any such lines.

Considering that I think that this feature is not very useful and that people are already complaining about Magit being to slow, I am afraid I have to say no.

**Comment by jazzzz** [comment]
Well, it makes me sad, because I love magit, but it's the only tool that don't let me know if there's a commit body, which is essential for reviewing merge/pull requests.

**Comment by jazzzz** [comment]
Wouldn't be possible to least add an option to allow this?

**Comment by tarsius** [comment]
> which is essential for reviewing merge/pull requests.

That sounds like a huge exaggeration. "It's convenient" sure, I can see that, but "essential"? Besides, when reviewing a pull request you have to review every commit anyway so you might as well review each commits message at the same time as you review its diff.

> Wouldn't be possible to least add an option to allow this?

No it would just be to much work for IMO to little gain.

**Comment by kyleam** [comment]
> but it's the only tool that don't let me know if there's a commit body, which is essential for reviewing merge/pull requests.

I suspect you already know, and it's just not close enough to the indicator that you want, but just in case: Are you aware of the log transient's `++header` flag and the associated `magit-log-revision-headers-format` option?  That'd let you see the full message without needing to move the cursor to each commit you're interested in.

**Comment by tarsius** [comment]
I was going to suggest that too but then I tried it out and it looked awful. Is it a regression that the graph is not drawn next to the commit body?

**Comment by kyleam** [comment]
> I was going to suggest that too but then I tried it out and it looked awful. 

Heh, true, it's not beautiful.

> Is it a regression that the graph is not drawn next to the commit body?

It smells like one, though I'm not positive.  I'm don't know the last time I've used `++header` myself.

**Comment by kyleam** [comment]
> > Is it a regression that the graph is not drawn next to the commit body?
> 
> It smells like one, though I'm not positive.

It is.  And I'm to blame :x

```
db97623afa6a22ec348724104d06c6ebf950d7b2 is the first bad commit
commit db97623afa6a22ec348724104d06c6ebf950d7b2
Author: Kyle Meyer <kyle@kyleam.com>
Date:   Sat Apr 15 00:27:39 2017 -0400
    magit-insert-{log,bisect-rest}: handle refs with parentheses
    
    magit-log-heading-re and magit-log-bisect-vis-re capture ref names
    with "[^()+]", but this fails when ref names include parentheses.
    
    Mark the end of the ref names field with NUL.
    
    Fixes #3028.
 Documentation/RelNotes/2.11.0.txt |  3 +++
 lisp/magit-bisect.el              |  2 +-
 lisp/magit-log.el                 | 10 +++++-----
```
